src/main/java/org/elasticsearch/kafka/indexer/jobs/ConsumerStartOption.java

```diff
@@ -118,4 +133,33 @@ public void setStartOffset(long startOffset) {
 		LATEST,
 		RESTART
 	}
+
+	@Override
+	public int hashCode() {
+		final int prime = 31;
```

Comment:
Please use this plugin [https://github.com/maximeAudrain/jenerate](https://github.com/maximeAudrain/jenerate) for Eclipse to generate more compact `hashCode` and `equals`. It's hard to look to this )